wdl/ExpansionHunterScatter.wdl

```diff
@@ -56,20 +57,21 @@ workflow ExpansionHunterScatter {
 
         call ExpansionHunter.ExpansionHunter  {
             input:
-                bam_or_cram=bam_or_cram_,
-                bam_or_cram_index=bam_or_cram_index_,
-                reference_fasta=reference_fasta,
-                reference_fasta_index=reference_fasta_index_,
-                split_variant_catalogs=SplitVariantCatalog.catalogs_json,
-                sample_id=sample_id,
-                ped_file=ped_file,
-                generate_realigned_bam=generate_realigned_bam,
-                generate_vcf=generate_vcf,
-                seeking_analysis_mode=seeking_analysis_mode,
-                expansion_hunter_docker=expansion_hunter_docker,
-                python_docker=python_docker,
-                runtime_eh=runtime_eh,
-                runtime_concat=runtime_concat
+                bam_or_cram = bam_or_cram_,
```

Comment:
I guess we have WDLs that format call parameter assignments differently scattered through the codebase (with varying amounts of whitespace around the =). I think no spaces might be the most common but I'm not sure about that. Fine to change it here for now if you prefer this style but we should probably eventually make a call on preferred style and format the whole codebase the same way. (this is just a note, not asking you to revert this change)